integration-tests/bats/sql-server.bats

```diff
@@ -552,4 +552,57 @@ SQL
      "
 
      server_query 1 "SELECT * FROM test" "pk,c1,c2,c3,c4,c5\n0,1,2,3,4,5\n1,1,2,3,4,5"
+}
+
+@test "sql-server: Run queries on database without ever selecting it" {
+     skiponwindows "Has dependencies that are missing on the Jenkins Windows installation."
+
+     cd repo1
+     start_sql_server repo1
```

Comment:
Make sure this is actually testing what you think it is. These test helper methods definitely `use` a particular database one way or another. 

Would be better to pass `repo2` here, since you're operating on repo1 below.

Reply:
So each of these queries form a new connection to the server. So if start_sql_server does in fact connect to the db, the unselected_sql_query will only connect to the serve but not the db. I can add an additional statement validating that no database is selected to give more clarity

Reply:
Just use repo2.one_pk etc. I know what you're doing is right, but you don't know how the underlying code might change and make your test worthless. If want the test to be durable, run the statements on a different database than the one you're giving as an argument to start_sql_server